recipes/gtest/all/patches/gtest-1.10.0-cmake.patch

```diff
@@ -0,0 +1,20 @@
+--- CMakeLists_old.txt	2021-05-03 07:32:54.443000000 +0000
++++ CMakeLists.txt	2021-05-03 07:33:27.422020100 +0000
+@@ -3,6 +3,17 @@
+ 
+ cmake_minimum_required(VERSION 2.8.8)
+ 
++include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
```

Comment:
Revert it. CMake wrapper is much simpler

Reply:
If the CMake wrapper is enabled not all unit test succeed. The reason is when you add a wrapper the binaries are located in a different folder and some python scripts of GTEST do not them anymore. That was the reason I solved that with a patch.